### PR TITLE
links section FE

### LINKED ISSUE #1486: Country Home/Links and repository (Files stored are accessible at country level - all cycles)
- Add table assessment_{assessmentName}.file 
 - add the business logic to area api/controller and file repository 
 - update ui, maybe as a list (to discuss)

### PR DESCRIPTION
**Pull request by rfontanarosa**
close #1486

https://user-images.githubusercontent.com/10047945/190120671-f3955c5f-2b30-4567-ab68-572510cf91de.mov

### CODE REVIEW

File: src/client/pages/AssessmentHome/FraHome/Links/Links.scss [hunk 1]
```diff
@@ -0,0 +1,56 @@
+@import 'src/client/style/partials';
+
+// .landing__link-container {
```
Comment (sorja) [inline src/client/pages/AssessmentHome/FraHome/Links/Links.scss:3]: this is not needed?

File: src/server/repository/assessment/file/remove.ts [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
   props: { assessment: Assessment; uuid: string },
   client: BaseProtocol = DB
-): Promise<{ fileName: string }> => {
+): Promise<AssessmentFile | undefined> => {
   const { assessment, uuid } = props
 
   const schemaName = Schemas.getName(assessment)
 
-  return client.one(`delete from ${schemaName}.file where uuid = $1 returning file_name;`, [uuid])
+  return client.one<AssessmentFile | undefined>(
```
Comment (sorja) [inline src/server/repository/assessment/file/remove.ts:15]: `oneOrNone`

**Review by sorja — CHANGES_REQUESTED** [review]
remove comments and use `oneOrNone`

File: src/server/repository/assessment/file/remove.ts [hunk 3]
```diff
@@ -1,14 +1,20 @@
...
   props: { assessment: Assessment; uuid: string },
   client: BaseProtocol = DB
-): Promise<{ fileName: string }> => {
+): Promise<AssessmentFile | undefined> => {
   const { assessment, uuid } = props
 
   const schemaName = Schemas.getName(assessment)
 
-  return client.one(`delete from ${schemaName}.file where uuid = $1 returning file_name;`, [uuid])
+  return client.oneOrNone<AssessmentFile | undefined>(
```
Comment (sorja) [inline src/server/repository/assessment/file/remove.ts:15]: No need undefined here

**Review by sorja — APPROVED** [review]

**Review by minotogna — APPROVED** [review]

**Pull request merged**